lib/Twig/Environment.php

```diff
@@ -448,6 +450,8 @@ public function resolveTemplate($names)
      */
     public function clearTemplateCache()
     {
+        @trigger_error(sprintf('%s is deprecated and will be removed in Twig 2.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Well, the documentation also says ``as of 1.x``, which does not make sense as we always introduced the replacement in a particular version